src/java.desktop/share/native/liblcms/LCMS.c

```diff
@@ -45,10 +46,9 @@
 
 #define SigHead TagIdConst('h','e','a','d')
 
-#define DT_BYTE     0
-#define DT_SHORT    1
-#define DT_INT      2
-#define DT_DOUBLE   3
+#define DT_BYTE     sun_java2d_cmm_lcms_LCMSImageLayout_DT_BYTE
+#define DT_SHORT    sun_java2d_cmm_lcms_LCMSImageLayout_DT_SHORT
+#define DT_INT      sun_java2d_cmm_lcms_LCMSImageLayout_DT_INT
```

Comment:
I don't understand why this isn't a step backwards.
It is overloading the "size" to mean the "type"
How would you tell the difference between SHORT vs UNSIGNED SHORT - both 2 bytes,
or INT vs FLOAT - both 4 bytes ?
or LONG vs DOUBLE ..
If we ever need to, then this all needs to be reverted.
% cd java/lang 
%  grep BYTES *java 
Byte.java:    public static final int BYTES = SIZE / Byte.SIZE;
Character.java:    public static final int BYTES = SIZE / Byte.SIZE;
Double.java:    public static final int BYTES = SIZE / Byte.SIZE;
Float.java:    public static final int BYTES = SIZE / Byte.SIZE;
Integer.java:    public static final int BYTES = SIZE / Byte.SIZE;
Long.java:    public static final int BYTES = SIZE / Byte.SIZE;
Short.java:    public static final int BYTES = SIZE / Byte.SIZE;